backend/internal/handlers/organizations/organization.go

```diff
@@ -123,3 +123,41 @@ func (service *OrganizationService) GetClassroomsInOrg() fiber.Handler {
 		})
 	}
 }
+
+func (service *OrganizationService) GetOrgTemplateRepos() fiber.Handler {
+	return func(c *fiber.Ctx) error {
+		// Extract org_id from the path
+		orgName := c.Params("org_name")
+		if orgName == "" || orgName == "undefined" {
+			log.Default().Println("Error getting org_name: ", orgName)
+			return c.Status(400).JSON(fiber.Map{"error": "invalid org_name"})
+		}
+
+		// Parse pagination parameters
+		itemsPerPage := c.QueryInt("itemsPerPage")
+		pageNum := c.QueryInt("pageNum")
+
+		// Get the user client
+		userClient, err := middleware.GetClient(c, service.store, service.userCfg)
+		if err != nil {
+			log.Default().Println("Error getting client: ", err)
+			return c.Status(500).JSON(fiber.Map{"error": "failed to create client"})
```

Comment:
We might want these to be custom errors in the errs package for future testing purposes I'd ask cam